adit/core/tasks.py

```
@@ -53,28 +55,54 @@ def backup_db(*args, **kwargs):
+
+def _cleanup_mass_transfer_exports(dicom_task: DicomTask) -> None:
+    if dicom_task._meta.app_label != "mass_transfer":
+        return
+
+    partition_key = getattr(dicom_task, "partition_key", None)
+    if not partition_key:
+        return
+
+    try:
+        from adit.mass_transfer.models import MassTransferVolume
+    except Exception:
```

Ruff (0.14.14)

[warning] Do not catch blind exception: `Exception`

(BLE001)